### ISSUE TITLE
`as_avatar: :square` option doesn't work

### ISSUE THREAD

**Issue by shiroemons** [avo-hq/avo#990]
### Describe the bug

Add `as_avatar: :square` option does not display square.

### Steps to Reproduce
1. Add `as_avatar: : square` option to image files
    - https://docs.avohq.io/2.0/search.html#avatar
    - > You may improve the results listing by adding an avatar to each search result. You do that by using the as_avatar attribute. This attribute has three options :square, :rounded or :circle. This influences the final roundness of the avatar.
2. Click on 'Search' form

### Expected behavior

Displayed as a circle

### Actual behavior

Displayed as a square

### System configuration
**Avo version**: 2.9.0

**Rails version**: 7.0.3

**Ruby version**: 3.1.2

**License type** (Community or Pro): Pro

### Additional context

Not implemented?
https://github.com/avo-hq/avo/blob/main/app/javascript/js/controllers/search_controller.js#L142-L150=

#### Impact

 - [ ] High impact
 - [ ] Medium impact
 - [x] Low impact

#### Urgency

 - [ ] High urgency
 - [ ] Medium urgency
 - [x] low urgency

**Comment by adrianthedev** [comment]
That's right! `:square` doesn't work. I added the `good first issue` label so someone can pick this up easily.

Thank you for digging into this!
